### PR TITLE
time correction in weather plugin

### PR DESCRIPTION
**Pull request by CallumCoy**
go even easier on me :sob:

### CODE REVIEW

File: src/app/plugins/weather.plugin.ts [hunk 1]
```diff
@@ -104,7 +104,7 @@ export class WeatherPlugin extends Plugin {
 
       if (hours > 0) {
         const dt = new Date(timestamp * 1000);
-        let strf: string = dt.getHours().toString();
+        let strf: string = (dt.getHours() % 12).toString();
```
Comment (tanndlin) [inline src/app/plugins/weather.plugin.ts:107]: Dont think this will work, this Gives a range from 0-11 o'clock

Comment (tanndlin) [reply src/app/plugins/weather.plugin.ts:107]: Make sure you test too

**Comment by CallumCoy** [comment]
oops forgot, solves #360

**Comment by tanndlin** [comment]
Not as simple as % 12, its other wise itll not change the AM/PM

**Comment by CallumCoy** [comment]
> Not as simple as % 12, its other wise itll not change the AM/PM

we'll see in 20 minutes, can't really test it manually, but I think it will work looking at how it has posted in the past

File: src/app/plugins/weather.plugin.ts [hunk 2]
```diff
@@ -104,7 +104,7 @@ export class WeatherPlugin extends Plugin {
 
       if (hours > 0) {
         const dt = new Date(timestamp * 1000);
-        let strf: string = dt.getHours().toString();
+        let strf: string = (dt.getHours() % 12 || 12).toString();
         strf += Math.floor(dt.getHours() / 12) < 1 ? 'AM' : 'PM';
```
Comment (c650) [inline src/app/plugins/weather.plugin.ts:108]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleTimeString

**Comment by CallumCoy** [comment]
> can we make `strf` const and better name?

Was const, the lint picked it up first :eyes:. as for the name sure, don't even know what it is meant to stand for.

**Review by tanndlin — APPROVED** [review]

**Pull request merged**